plugin/completion.py

```diff
@@ -41,10 +42,62 @@
 
 
 class LspResolveDocsCommand(sublime_plugin.TextCommand):
+    def run(self, edit: sublime.Edit, index: int) -> None:
+        item = CompletionHandler.completions[index]
+        detail = minihtml(item.get('detail') or "", self.view)
+        documentation = minihtml(item.get("documentation") or "", self.view)
+
+        # don't show the detail in the cooperate AC popup if it is already shown in the AC details filed.
+        self.is_detail_shown = bool(detail)
+
+        content = self.get_content(documentation, detail)
+        self.show_popup(content)
+
+        if not detail or not documentation:
+            # To make sure that the detail or documentation fields doesn't exist we need to resove the completion item.
+            # If those fields appear after the item is resolved we show them in the popup.
+            self.do_resolve(item)
+
+    def get_content(self, documentation: str, detail: str) -> str:
+        content = ""
+        if detail and not self.is_detail_shown:
+            content += "<div class='highlight' style='margin: 4px'>{}</div>".format(detail)
+
+        if documentation:
+            content += "<div style='margin: 10px'>{}</div>".format(documentation)
+
+        return content
+
+    def show_popup(self, content: str) -> None:
+        mdpopups.show_popup(
```

Comment:
I don't think this is correct? Is it always interpreted as markdown now? Or is `md=True` needed to make that happen?

What happens when we pass minihtml in here?

Reply:
I don't think we need to set `md=True`. because we already converted the content if from markdown to html(if it was markdown) in minihtml with md2html. 

I will rename `content` to minihtml.